doc/tutorial/Authentication.lhs

```diff
@@ -318,7 +318,7 @@ genAuthAPI = Proxy
 
 Now we need to bring everything together for the server. We have the
 `AuthHandler Request Account` value and an `AuthProtected` endpoint. To bind these
-together, we need to provide a [Type Family](https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/type-families.html)
+together, we need to provide a [Type Family](https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/glasgow_exts.html#type-families)
```

Comment:
I'd link to specific GHC manual, i.e. latest -> 8.8.1, otherwise that link will die sooner or later (especially anchor might be volatile).

Or what @alpmestan thinks?